docs/public-offer/index.rst

```diff
@@ -113,12 +113,15 @@ Once offer field is ready, run the command:
 
          awsmp public-offer update-instance-type \
             --product-id prod-xwpv7txqxg55e \
-            --instance-type-file listing_configuration.yaml \
-            --dimension-unit Hrs
+            --config listing_configuration.yaml \
+            --dimension-unit Hrs \
+            --price-change-allowed N
```

Comment:
I don't think you need to apply N (assuming above changes are made). If the flag is in the command invocation then it should set the book to true. Otherwise, the bool is false. The UX feels a little off if the user has to supply it if they want to fail on changes